deps/wazuh_testing/wazuh_testing/data_visualization_module/README.md

```diff
@@ -0,0 +1,141 @@
+
+Another example of a configuration file:
+
+```shell script
+Component:
+  - "dashboard"
+
+Processes:
+  Item:
+    - "wazuh-dashboard"
+
+Columns_to_avoid:
+  - "Daemon"
+  - "Version"
+  - "Timestamp"
+  - "Commit"
+```
+
+This means that the visualization is not only valid for Wazuh daemons, but also for other types of processes.
+
+### CSV format
+
+The CSV file name must match the value in the `Version` column. This is to improve the efficiency of database queries. For example: `4.8.0-beta4.csv`. In addition to the `Version` column, it should contain another column named `Commit`, indicating the commit in which the file was obtained (e.g. nightly, weekly, pre-release, etc.), and a column with the process to be displayed, which can be the wazuh daemons, the wazuh-dashboard itself, etc. The name of this same column must match the name specified in the YAML file. The rest of the columns may be informative columns, of type string, which must be included in `Columns_to_avoid` in the YAML file, and data columns containing the metrics taken.
+
+Considering the two examples of YAML files above, the CSV files could be respectively:
+
+```shell script
+Version,Commit,Timestamp,Daemon,CPU(%),RSS(KB),VMS(KB),FD,Read_Ops,Write_Ops,Disk_Read(B),Disk_Written(B),Disk(%),USS(KB),PSS(KB),SWAP(KB)
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-syscheckd,0.0,8188.0,76972.0,7,20,21,0,49152,0.00019935213463462376,2720.0,3819.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-analysisd,0.0,34944.0,1159264.0,24,1766,37,0,57344,0.00023257749040706108,32412.0,32720.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-maild,0.0,0.0,0.0,0,0,0,0,0,0,0.0,0.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-execd,0.0,3772.0,46484.0,5,11,3,0,8192,3.3225355772437296e-05,1740.0,1831.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-modulesd,0.0,112060.0,584528.0,132,28063,41097,90960896,300437504,1.5874452012649805,100996.0,104033.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-clusterd,0.0,0.0,0.0,0,0,0,0,0,0,0.0,0.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-db,0.0,18448.0,959756.0,40,6738,931,5345280,2187264,0.03055071463275609,10996.0,12981.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-remoted,0.0,20204.0,1246528.0,39,5166,2097,0,57344,0.00023257749040706108,13092.0,14940.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-integratord,0.0,0.0,0.0,0,0,0,0,0,0,0.0,0.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-monitord,0.0,4080.0,55424.0,8,13,2,0,4096,1.6612677886218648e-05,1960.0,2056.0,0.0
+4.8.1,pre-release,2024-07-05 21:12:27,wazuh-authd,0.0,7900.0,136788.0,7,16,4,8192,4096,4.983803365865594e-05,2096.0,3033.0,0.0
```

Comment:
The removal of repetitive values like commit or version could be a good idea in order to improve performance and make better use of the space database.
In the future we need to integrate this information during the construction of the database,

Reply:
We will handle it in a data standardization and database improvement issue